.circleci/config.yml

```
@@ -125,8 +122,7 @@ commands:
             curl -LsSf https://astral.sh/uv/install.sh | sh
             uv venv
             source .venv/bin/activate
-            uv pip install .
-            uv pip install -r ./test_requirements/requirements_optional.txt
+            uv pip install .[dev_optional]
```

For consistency, this should use `uv sync`, right?

```suggestion
            uv sync --extra dev_optional
```